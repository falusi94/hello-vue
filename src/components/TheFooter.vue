<template>
  <div class="footer">
    <div class="footer-section">
      Made with â™¥ and
      <a target="_blank" href="https://vuejs.org">Vue</a>
      by
      <a target="_blank" href="https://github.com/falusi94">@falusi94</a>
    </div>
    <div class="footer-section">
      Data is from the <a target="_blank" href="https://punkapi.com">Punk API</a>
    </div>
  </div>
</template>

<script>
export default {
  name: 'TheFooter',
};
</script>

<style lang="scss" scoped>
  .footer {
    height: 4rem;
    background-color: #ECEFF1;
    text-align: center;
    padding: 2rem;

    &-section {
      margin-bottom: 0.5rem;
    }

    a, a:link, a:visited, a:hover, a:active {
      text-decoration: none;
      font-weight: 500;
    }

    a, a:link {
      color: #607D8B;
    }

    a:visited {
      color: #455A64;
    }

    a:hover, a:active {
      color: #263238;
      transition-duration: 0.5s;
    }
  }
</style>
