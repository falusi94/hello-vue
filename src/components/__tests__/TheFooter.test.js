import { shallowMount } from '@vue/test-utils';
import TheFooter from '../TheFooter.vue';

describe('TheFooter.vue', () => {
  it('renders properly', () => {
    const wrapper = shallowMount(TheFooter);
    expect(wrapper).toMatchSnapshot();
  });
});
