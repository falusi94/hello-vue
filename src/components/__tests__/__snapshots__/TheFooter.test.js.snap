// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheFooter.vue renders properly 1`] = `
<div class="footer">
  <div class="footer-section">
    Made with â™¥ and
    <a target="_blank" href="https://vuejs.org">Vue</a>
    by
    <a target="_blank" href="https://github.com/falusi94">@falusi94</a>
  </div>
  <div class="footer-section">
    Data is from the <a target="_blank" href="https://punkapi.com">Punk API</a></div>
</div>
`;
